src/components/header/Header.tsx

```diff
@@ -34,6 +39,7 @@ const Header = (props: Props) => {
       </Link>
       <Nav className="me-auto">
         <AsyncTypeahead
+          key={id}
```

Comment:
I found in documentation that we can't use `useId` to generate keys in a list. https://react.dev/reference/react/useId